ui/src/components/DetailListCard.svelte

```diff
@@ -0,0 +1,90 @@
+<script>
+  import CodeSummary from "./CodeSummary.svelte";
+  import LinkSummary from "./LinkSummary.svelte";
+  import LighthouseSummary from "./LighthouseSummary.svelte"; 
+  import ArtillerySummary from  "./ArtillerySummary.svelte";
+  import formatDistanceToNow from "date-fns/formatDistanceToNow";
+  import { format } from 'date-fns';
+  import { printTimeDiff } from "../utils/utils";
+  import { Navigate, navigateTo } from "svelte-router-spa";
+  import ArtilleryDetailTable from "./ArtilleryDetailTable.svelte";
+  export let value = {};
+  </script>
+
+  <style>
+  h2 {
+   text-align: left; 
+   border-bottom: 1px dotted #000; 
+   line-height: 0.1em;
+   margin-bottom: 15px; 
+  }
+
+  h2 span { 
+    background:#fff; 
+    padding-left: 0px; 
+    padding-right: 10px; 
+  }
+
+  .container {
+  transition: 0.3s;
+  }
+
+  .container:hover {
+  box-shadow: 0 8px 16px 0 rgba(0,0,0,0.2);
+  cursor: pointer;
+  }
+  </style>
+
+  
+  {#each value as val}
+    <div class="container">
+      <div class="overflow-hidden shadow-lg">
+
+        {#if val.finalEval == "FAIL"}
```

Comment:
in JS, always use `===` instead of `==` 

see https://eslint.org/docs/rules/eqeqeq